SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -113,27 +113,17 @@ Interest InterestFactory::CreateServerWorkerInterest()
 
 Interest InterestFactory::CreateInterest() const
 {
-	if (GetDefault<USpatialGDKSettings>()->bEnableServerQBI)
+	if (Actor->IsA(APlayerController::StaticClass()))
```

Comment:
Only add player interest once for each player. With the ever increasing size of the interest component, this will hopefully be a saving even if server QBI is enabled.

Reply:
Importantly, this interest component will only be sent over the wire to the client authoritative over the entity, so this PR will also introduce less sending interest components around full stop